togetherly/lib/views/widgets/new_chore_dialog.dart

```diff
@@ -100,14 +160,45 @@ class _NewChoreDialogState extends State<NewChoreDialog> {
   void toggleWeekday(int i) =>
       setState(() => _repeatingWeekdays[i] = !_repeatingWeekdays[i]);
 
+  Widget buildDropdown() {
+    return SingleChildScrollView(
+      child: SizedBox(
+        width: MediaQuery.of(context).size.width * 0.45,
+        child: MultiSelectDropDown<Child>(
+          controller: selectController,
+          selectionType: SelectionType.multi,
+          dropdownHeight: 200,
+          chipConfig: const ChipConfig(
+            wrapType: WrapType.scroll,
+            backgroundColor: AppColors.brandLightGray,
+            labelColor: AppColors.brandWhite,
+            labelStyle: AppTextStyles.brandBodySmall,
+          ),
+          clearIcon: const Icon(
+            Icons.clear,
+            color: AppColors.brandBlack,
+          ),
+          onOptionSelected: (options) {},
+          selectedOptions: selectController.selectedOptions,
```

Comment:
Is this how the widget is meant to be used? It seems like it might be redundant to pass the dropdown both a controller and the list of options from the controller. Then again, it is a third-party library, so maybe that's just how they've designed it.

Reply:
I'll fiddle around with it. It's possible it could be simplified.

Reply:
After looking at it, the only parameter I could remove is selectedOptions. That one is optional and is covered when the controller is set in the initState. I'll leave it commented for now in case there's a reason we need it later.